packages/studio-ui/src/components/UndoRedo.tsx

```diff
@@ -22,6 +22,20 @@ export default function UndoRedo(): JSX.Element {
     redo();
   }, [redo]);
 
+  const handleUndoKeydown = useCallback(
+    (event: KeyboardEvent) => {
+      if ((event.ctrlKey || event.metaKey) && event.key === "z") {
```

Comment:
Talked with Alex offline. He's going to investigate the `Platform.js` library. This library supposedly can tell us which OS a browser is being used with. It's a very popular, stable library. If it works, we can be more precise here and not set two key bindings on both MacOS and Windows.

Reply:
Works locally and in Storm! Very cool-- now, the shortcut uses command if the user OS is OS X, and defaults to control if not.